examples/json_field.rs

```diff
@@ -53,6 +51,8 @@ fn main() -> tantivy::Result<()> {
     let reader = index.reader()?;
     let searcher = reader.searcher();
 
+    // # Default fields: event_type and attributes
+    // By setting attributes as a default field it allows omitting attributes itself, e.g. "target", instead of "attributes.target"
```

Comment:
```suggestion
    // By setting attributes as a default field it allows omitting attributes itself, e.g. "target",
    // instead of "attributes.target"
```